config/core/test-configs-chromeos.yaml

```diff
@@ -790,6 +790,22 @@ device_types:
         modules: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-sarien/20231106.0/amd64/modules.tar.xz'
       block_device: detect
 
+  hp-14-db0003na-grunt_chromeos:
+    <<: *chromebook-x86-type
```

Comment:
Maybe it would be interesting to have a `chromebook-grunt-type` ref? we have 3 of those boards with this commit, all sharing the same exact parameters, except for `base_name`.